### PR TITLE
cleaned out target links that no longer work

### PR DESCRIPTION
**Pull request by mark-b-kauffman**

### CODE REVIEW

**Comment by mark-b-kauffman** [comment]
6. Dev pushes a PR based on their branch <-- just did this.
7. System receives PR and does a test build <-- when does this happen? do i need to wait for approval?
8. If successful the PR may be merged and the branch deleted

**Review by daveyherrera — APPROVED** [review]
noticed that nothing that was changed breaks anything.

**Pull request merged**
